exercises/rotational-cipher/example.py

```diff
@@ -0,0 +1,12 @@
+def rotational_cipher(message, key):
+    import string
+    alpha_lower = list(string.ascii_lowercase)
+    alpha_upper = list(string.ascii_uppercase)
```

Comment:
Instead of creating the alpha variables, you could rename them inside the import statement like this:
```python
from string import ascii_lowercase as alpha_lower
from string import ascii_uppercase as alpha_upper
``